test/hotspot/jtreg/compiler/vectorapi/TestVectorTest.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2022, 2025,Oracle and/or its affiliates. All rights reserved.
```

Comment:
The jcheck is complaining about a copyright format issue here. There should be a space after `2025,`.

Reply:
Thanks! Let me fix it.